### PR TITLE
Package diagnostics

### PR DESCRIPTION
**Pull request by JJJianJiao**
I finish the Package-Diagnostics task.

Please review my work. 

Thanks very much!

PS: I feel confused with the Experimental and Preview in unity. I need to discuss this with Marco later.

### CODE REVIEW

**Review by justinkiffiak-SM — COMMENTED** [review]
Nice Work. Just a tweak or two regarding spelling, along with Unit Tests, and page created in Documentation ( manual ).

**Review by justinkiffiak-SM — COMMENTED** [review]
Looks great, now onto the Unit Tests and Documentation page.

File: Editor/UI/ProjectAuditorWindow.cs [hunk 1]
```diff
@@ -347,6 +347,16 @@ static void OnLoad()
...
                 });
 
+                ViewDescriptor.Register(new ViewDescriptor
+                {
+                    category = IssueCategory.PackageVersion,
+                    name = "Packages Version",
+                    menuLabel = "Experimental/Packages Version",
+                    menuOrder = 106,
+                    showRightPanels = true,
+                    analyticsEvent = (int)ProjectAuditorAnalytics.UIButton.Packages
```
Comment (mtrive) [inline Editor/UI/ProjectAuditorWindow.cs:357]: this should be _PackageVersion_

**Review by mtrive — CHANGES_REQUESTED** [review]

File: Editor/Modules/PackagesModule.cs [hunk 2]
```diff
@@ -31,6 +35,36 @@ class PackagesModule : ProjectAuditorModule
                 new PropertyDefinition { type = PropertyTypeUtil.FromCustom(PackageProperty.Source), format = PropertyFormat.String, name = "Source", defaultGroup = true }
             }
         };
+
+        static readonly IssueLayout k_PackageVersionLayout = new IssueLayout
+        {
+            category = IssueCategory.PackageVersion,
+            properties = new[]
+            {
+                new PropertyDefinition { type = PropertyType.Description, name = "Display Name"},
```
Comment (mtrive) [inline Editor/Modules/PackagesModule.cs:44]: let's rename to _Name_

File: Editor/Modules/PackagesModule.cs [hunk 3]
```diff
@@ -31,6 +35,36 @@ class PackagesModule : ProjectAuditorModule
...
+                new PropertyDefinition { type = PropertyTypeUtil.FromCustom(PackageVersionProperty.Name), format = PropertyFormat.String, name = "Package Name", defaultGroup = true},
```
Comment (mtrive) [inline Editor/Modules/PackagesModule.cs:45]: let's use _Id_ and _Package Id_ for _name_  and _longName_ respectively

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/ProjectAuditor?pullRequest=114) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/ProjectAuditor?pullRequest=114) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/ProjectAuditor?pullRequest=114) it.</sub>

File: Editor/UI/ProjectAuditorWindow.cs (hunk 1 above)
Comment (mtrive) [reply Editor/UI/ProjectAuditorWindow.cs:357]: Have you changed this to use ProjectAuditorAnalytics.UIButton.PackageVersion ?

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/ProjectAuditor?pullRequest=114) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/ProjectAuditor?pullRequest=114) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/ProjectAuditor?pullRequest=114) it.</sub>

**Pull request closed without merging**